frontend/src/components/find-replace/find-replace.tsx

```diff
@@ -0,0 +1,154 @@
+            value={state.findText}
+            autoFocus={true}
+            className="mr-2 mb-0"
+            placeholder="Find"
+            onKeyDown={(e) => {
+              if (e.key === "Enter") {
+                if (e.shiftKey) {
+                  findPrev();
+                } else {
+                  findNext();
+                }
+              }
+            }}
+            onChange={(e) => {
+              dispatch({ type: "setFind", find: e.target.value });
+            }}
+          />
+          <Input
+            value={state.replaceText}
+            placeholder="Replace"
+            onChange={(e) => {
+              dispatch({ type: "setReplace", replace: e.target.value });
+            }}
+          />
+        </div>
+        <div className="flex flex-col gap-2">
+          <div className="flex items-center gap-[2px]">
+            <Tooltip content="Case Sensitive">
+              <Toggle
+                size="sm"
+                pressed={state.caseSensitive}
+                data-state={state.caseSensitive ? "on" : "off"}
+                onPressedChange={(pressed) =>
+                  dispatch({ type: "setCaseSensitive", caseSensitive: pressed })
+                }
+              >
+                <CaseSensitiveIcon className="w-4 h-4" />
+              </Toggle>
+            </Tooltip>
+            <Tooltip content="Use Regular Expression">
+              <Toggle
+                size="sm"
+                pressed={state.regexp}
+                data-state={state.regexp ? "on" : "off"}
+                onPressedChange={(pressed) =>
+                  dispatch({ type: "setRegex", regexp: pressed })
+                }
+              >
+                <RegexIcon className="w-4 h-4" />
+              </Toggle>
+            </Tooltip>
+          </div>
+          <div className="flex items-center gap-[2px]">
+            <Button
+              size="xs"
+              variant="outline"
+              className="h-6 text-xs"
+              onClick={() => replaceNext()}
+              disabled={state.findText === "" || state.replaceText === ""}
+            >
+              Replace Next
+            </Button>
+            <Button
+              size="xs"
+              variant="outline"
+              className="h-6 text-xs"
+              onClick={() => replaceAll()}
+              disabled={state.findText === "" || state.replaceText === ""}
+            >
+              Replace All
+            </Button>
+          </div>
+        </div>
+      </div>
+      <div className="flex items-center gap-2">
+        <Tooltip content="Find Next">
+          <Button size="xs" variant="secondary" onClick={() => findNext()}>
+            <ArrowLeftIcon className="w-4 h-4" />
+          </Button>
+        </Tooltip>
+        <Tooltip content="Find Previous">
+          <Button size="xs" variant="secondary" onClick={() => findPrev()}>
+            <ArrowRightIcon className="w-4 h-4" />
+          </Button>
+        </Tooltip>
```

Comment:
Let's switch these: `ArrowLeftIcon` should be `findPrev` and `ArrowRightIcon` should be `findNext`. I think that's more natural/standard.

Reply:
oops, yea thats wrong